test/gtest/ucp/test_ucp_peer_failure.cc

```diff
@@ -287,6 +289,23 @@ void test_ucp_peer_failure::do_test(size_t msg_size, int pre_msg_count,
 
     EXPECT_EQ(UCS_OK, m_err_status);
 
+#if HAVE_IB
+    /* Since UCT/UD EP has a SW implementation of reliablity on which peer
+     * failure mechanism is based, we should set small UCT/UD EP timeout
+     * for UCT/UD EPs for sender's UCP EP to reduce testing time */
+    ucp_ep_config_t *ep_config = ucp_ep_config(failing_sender());
+    for (ucp_lane_index_t lane = 0; lane < ep_config->key.num_lanes; ++lane) {
+        ucp_rsc_index_t rsc_index  = ep_config->key.lanes[lane].rsc_index;
+        if (rsc_index != UCP_NULL_RESOURCE) {
+            ucp_worker_iface_t *wiface = ucp_worker_iface(sender().worker(), rsc_index);
+            if (wiface->iface->ops.iface_flush == uct_ud_iface_flush) {
```

Comment:
minor: i would check for uct_ud_iface_handle_failure instead of flush, and also comment that we are doing this to check if the iface is ud transport
and include ud_iface.h

Reply:
discussed offline
we will go with `uct_ud_iface_flush` as it is now